diabetes_regression/azureml_environment.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "diabetes_regression_sklearn",
+    "version": null,
```

Comment:
where is this file used?

Reply:
In Environment.load_from_directory(). Added comment to script for clarity https://github.com/microsoft/MLOpsPython/pull/168/commits/fe35f831ee0add4139f671dcba30e45e519e8e3b

Reply:
azureml_environment.json is not pointing to any conda specification file

Reply:
That is the file as generated by `az ml environment scaffold`.

I assume your concern is about the `null` values in this section of the file:

```
    "python": {
        "userManagedDependencies": false,
        "interpreterPath": "python",
        "condaDependenciesFile": null,
        "baseCondaEnvironment": null
    },
```

If you look at the AML SDK source code, load_from_directory() actually fills the conda_dependencies from a file named `conda_dependencies.yml` is one exists in the passed directory: 

```
    def load_from_directory(path):
        """Load an environment definition from the files in a directory.

        :param path: Path to the source directory.
        :type path: str
        """
        definition_path = os.path.join(path, _DEFINITION_FILE_NAME)
        if not os.path.isfile(definition_path):
            raise FileNotFoundError(definition_path)

        with open(definition_path, "r") as definition:
            environment_dict = json.load(definition)
        env = Environment._deserialize_and_add_to_object(environment_dict)

        conda_file_path = os.path.join(path, _CONDA_DEPENDENCIES_FILE_NAME)
        if os.path.isfile(conda_file_path):
            env.python = env.python or PythonSection(_skip_defaults=True)
            env.python.conda_dependencies = CondaDependencies(conda_file_path)

        base_dockerfile_path = os.path.join(path, _BASE_DOCKERFILE_FILE_NAME)
        if os.path.isfile(base_dockerfile_path):
            with open(base_dockerfile_path, "r") as base_dockerfile:
                env.docker = env.docker or DockerSection(_skip_defaults=True)
                env.docker.base_dockerfile = base_dockerfile.read()

```

I've tested that things works as expected, as when I uncomment `r-essentials` from the conda file and run `ml_service/pipelines/diabetes_regression_build_train_pipeline_with_r.py`, training succeeds as shown by R output in the logs.